app/build.gradle

```diff
@@ -84,6 +78,14 @@ android {
       }
     }
 
+  applicationVariants.all { variant ->
```

Comment:
This follows indentation rules defined in editor config. Please let me know if it needs to be aligned with the existing style.